tests/runtest.py

```diff
@@ -801,6 +810,8 @@ def parse_argument():
                         help="Parallel worker count; using all core for default")
     parser.add_argument("-c", "--compiler", dest='compiler', default="gcc",
                         help="Select compiler gcc or clang. (gcc by default)")
+    parser.add_argument("-k", "--keep", dest='keep', action='store_true',
```

Comment:
Please change this to `store_false`.

Reply:
in my environment, changing 'store_true' to 'store_false' 
cause not keeping test directory when using option -k,  but keeping test directory when not using option -k.

Reply:
Ah, right. Please ignore this and keep using `store_true`.